src/benchmarks/micro/libraries/System.Runtime/Perf.Int16.cs

```diff
@@ -32,5 +32,11 @@ public class Perf_Int16
         [Benchmark]
         [ArgumentsSource(nameof(Values))]
         public string ToString(short value) => value.ToString();
+
+#if NET7_0_OR_GREATER
+        [Benchmark]
+        [Arguments(1, -1)]
```

Comment:
Shouldn't this test copying both a positive and negative sign?

The actual code, due to it being two's complement, has different costs based on whether `value` is positive or negative and whether `sign` is positive or negative.

Reply:
> Shouldn't this test copying both a positive and negative sign?

I tried that and the reported time was almost identical for all test cases (like 0.4 ns for Int16)